cmake_modules/juce_helpers.cmake

```diff
@@ -38,6 +38,10 @@ function(add_juce_vst3_plugin PLUGIN_NAME)
   target_include_directories(${PLUGIN_NAME}_VST3 PRIVATE ${_SUPPORT_PATH}/ ${EPS_SHARED_DIR})
   target_link_libraries(${PLUGIN_NAME}_VST3 PRIVATE Juce::VST3)
 
+  set(EPS_PLUGIN_TARGETS_COPY ${EPS_PLUGIN_TARGETS})
+  list(APPEND EPS_PLUGIN_TARGETS_COPY ${PLUGIN_NAME}_VST3 )
+  set(EPS_PLUGIN_TARGETS ${EPS_PLUGIN_TARGETS_COPY} PARENT_SCOPE)
+
```

Comment:
Yeah that would be a lot cleaner! The only problem is it would include ADM Export Source VST which actually wants copying separately - at the moment I just don't pass EPS_PLUGIN_TARGETS back up the chain after the helper func has run for that target, which omits it from the list. I can work around that though.